platforms/hyperledger-fabric/configuration/roles/create/configtx/templates/configtxOrderer.tpl

```diff
@@ -2,11 +2,11 @@ Orderer: &OrdererDefaults
   OrdererType: {{ consensus.name }}
   Addresses:
 {% for orderer in orderers %}
-  {%- if provider == 'minikube' %}
+{% if provider == 'minikube' %}
```

Comment:
removing - will keep newlines. Is this needed?

Reply:
the list looks better with new lines
earlier it was like 
addresses:
  - a.com   - b.com   -c.com

now it is 
addresses:
  - a.com
  - b.com
  - c.com